ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/representative-journey/representative-journey.ts

```diff
@@ -86,23 +88,19 @@ export class RepresentativeJourney extends JourneyBase {
 
     let nextStep = this.stepOrder[currentStep + 1];
 
-    // incase of 'no' response for access to computer and camera navigate to questionnaire completed, contact us
-    // access to a computer.
-    if (this.model.computer === false) {
-      if (this.currentStep === RepresentativeJourneySteps.QuestionnaireCompleted) {
-        nextStep = RepresentativeJourneySteps.ContactUs;
-      } else {
-        this.submit();
-        nextStep = RepresentativeJourneySteps.QuestionnaireCompleted;
-      }
+    if (this.submitService.isDropOffPoint(this.model) && !this.isSubmitted) {
+      // update the model to set the answers in case browserback was clicked and the answers were changed.
+      let saveModel: MutableRepresentativeSuitabilityModel;
```

Comment:
I think you can inline this to just be:
`const model = this.submitService.updateSubmitModel(this.currentStep, this.model)`

Reply:
This call is not required so removed the code